examples/README.md

```diff
@@ -0,0 +1,171 @@
+- The resources available for the Flink Deployment
+
+The Flink Deployment configuration does __NOT__ contain the following:
+- The job to run
+- Any job specific configurations
+
+To create a Flink Deployment with the specific resources without any jobs run the following command:
+```bash
+kubectl apply -f basic-session-deployment-only.yaml
+```
+
+##### Adding jobs
+
+For an existing Flink Deployment another configuration could be used to create new jobs.
+This configuration should contain the following:
+- The Flink Deployment to use
+- The job to run
+- Any job specific configurations
+
+If the Flink Deployment is created by `basic-session-deployment-only.yaml` new job could be added
+by the following command:
+```bash
+kubectl apply -f basic-session-job-only.yaml
+```
+
+#### Creating Deployment and Jobs together
+
+Alternatively the Flink Deployment and the Flink Session Job configurations can be submitted together.
+
+To try out this run the following command:
+```bash
+kubectl apply -f basic-session-deployment-and-job.yaml
+```
+
+### SQL runner
+
+For running Flink SQL scripts check this [example](flink-sql-runner-example/README.md).
+
+### Python example
+
+For running Flink Python jobs check this [example](flink-python-example/README.md).
+
+### Advanced examples
+
+There are typical requirements for production systems and the examples below contain configuration files
+showing how to archive some of these.
+
+#### Custom logging
+
+This example adds specific logging configuration for the Flink Deployment using the
+`logConfiguration` property. To try out this run the following command:
+```bash
+kubectl apply -f custom-logging.yaml
+```
+
+#### Pod templates
+
+Flink Kubernetes Operator provides the possibility to simplify the deployment descriptors by using
+[Pod Templates](https://nightlies.apache.org/flink/flink-kubernetes-operator-docs-main/docs/custom-resource/pod-template/).
+
+This example shows how these templates are created and used. To try out this run the following command:
+```bash
+kubectl apply -f pod-template.yaml
+```
+
+#### Ingress
+
+Flink's Web UI access can be configured by the
+[Ingress](https://nightlies.apache.org/flink/flink-kubernetes-operator-docs-main/docs/operations/ingress/)
+entries.
+
+##### Basic Ingress example
+
+Simple domain based ingress configuration could be tried out by running this command:
+```bash
+kubectl apply -f basic-ingress.yaml
+```
+
+##### Advanced Ingress example
+
+It is possible to generate path based routing. To try out this run the following command:
+```bash
+kubectl apply -f advanced-ingress.yaml
+```
+
+#### High availability
+
+Basic example to configure Flink Deployments in
+[HA mode](https://nightlies.apache.org/flink/flink-docs-release-1.15/docs/deployment/ha/overview/).
+The example shows how to set savepoint directory, checkpoint directory and HA. To try out this run the following command:
+```bash
+kubectl apply -f basic-checkpoint-ha.yaml
+```
```

Comment:
I would not call this "advanced" as this is a very general requirement. We should have this near the top and also I would prefer to call it `Checkpointing & High Availability`

Reply:
Thanks for the insight!
Done